hangman_edited_1

```diff
@@ -0,0 +1,141 @@
+# Name:
```

Comment:
Thực ra thì H ko cần phải tạo một file mới. Tốt nhất nên viết chồng lên file cũ vì:
1. Phiên bản cũ đã được lưu bởi git ko sợ mất nên ko lo chuyện mất code.
2. Diff sẽ dễ xem và review hơn (thay vì thấy toàn bộ là file mới).
3. Trong dự án thật, người ta thay đổi file chứ không tạo file mới.